schemas/data/datasource.schema.json

```diff
@@ -9,15 +9,15 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "Data Source",
   "type": "object",
-  "description": "The Datasource acts as a namespace or unique identifier associated with a collection of data. Each EndUserID is associated with a given Datasource. A user can create and obtain information about a Datasource during the solution onboarding.\n",
+  "description": "The Data source acts as a namespace or unique identifier associated with a collection of data. Each EndUserID is associated with a given data source. A user can create and obtain information about a data source during the solution onboarding.\n",
   "definitions": {
```

Comment:
Data source or data source?

Reply:
Definitely data source in this context.  You have an eagle eye!